t/scripts/create_test_db.py

```diff
@@ -30,7 +31,7 @@ def edit_usage_col(conn, username, value):
 
 
 def main():
-    filename = "/usr/src/t/expected/t_small_no_tie.db"
+    filename = sys.argv[1]
```

Comment:
To be on the safe side, it'd be a good idea to check that the user input `sys.argv[1]`.  Perhaps output an error if the user didn't.

Reply:
Thanks for this suggestion; I just squashed and force-pushed a change that performs a simple check that the user input `sys.argv[1]`.